packages/preact-router/src/useBlocker.tsx

```
@@ -0,0 +1,322 @@
+import {
+  useContext,
```

ESLint

[error] 'useContext' is defined but never used.

(unused-imports/no-unused-imports)